src/renderer/components/ContextMenuSearchSettings/ContextMenuSearchSettings.css

```
@@ -0,0 +1,85 @@
+.engineRow {
+  display: grid;
+  grid-template-columns: minmax(160px, 1fr) minmax(240px, 2fr);
+  gap: 12px;
+  align-items: center;
+}
+
+.engineRow:has(.removeEngine) {
+  grid-template-columns: minmax(160px, 1fr) minmax(140px, 1fr) minmax(240px, 2fr) auto;
+}
+
+:deep(.engineRow .ft-input),
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)